bundle/DependencyInjection/Configuration/Parser/LimitationValueTemplates.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
```

Comment:
Please add license info PhpDoc

Reply:
Just run `php-cs-fixer`. It should add the missing header.